dprint.json

```diff
@@ -18,7 +18,10 @@
     "maker-frontend/dist",
     "taker-frontend/dist",
     "**/node_modules",
-    "rocket-cookie-auth/example-http-cookie-auth.sql"
+    "rocket-cookie-auth/example-http-cookie-auth.sql",
```

Comment:
why make this special snowflake? is there a reason why it shouldn't be formatted consistently too? (also: it's completely unrelated to the PR)

Reply:
Actually this file does not exist anymore...

Reply:
then we can remove the special case, problem solved eitherway :-)